### ISSUE TITLE
Multi-finger exploration

### ISSUE THREAD

**Issue by nawidj** [tacman-fp7/object-surface-exploration#4]
# Single finger vs multi finger exploration
Evaluating the benefit of the additional finger on the exploration.

## Method
Use two objects of different size to check whether the additional finger can help in speeding up the exploration, hence, the surface reconstruction.

## Objects
Left: small object, right: large object
![objects](https://cloud.githubusercontent.com/assets/10450269/20143554/a42ac516-a699-11e6-8767-1d6e933d478b.png)

### Small object
![smallobjectresults](https://cloud.githubusercontent.com/assets/10450269/20143624/f0c61600-a699-11e6-91e8-de33482f7a06.png)

### Large object
![largeobjectresults](https://cloud.githubusercontent.com/assets/10450269/20143638/08bd7f0a-a69a-11e6-9e3f-59f650a90559.png)

## Discussion

Initial results on small objects is not conclusive. There seems to be some anomalies in the data for the multiple finger data, which is making the results look worse. This needs to be investigated

On large objects the results indicate that the additional fingertip is beneficial. Further analysis is needed to confirm if that is that case.

**Comment by lornat75** [comment]
Maybe with a few more points the advantage for the large object would become more evident, at the moment is really difficult to tell the difference between right and left meshes.

**Comment by nawidj** [comment]
I agree, I have a couple of data collection issues to iron out. I will collect another set for both cases on Friday.

**Comment by nawidj** [comment]
Plot of the data collected today. Neither approach -- single finger or multi-finger -- managed to capture the shape very well. I save the fingertip positions separately now. Plus, for this dataset I have collected enough information to study the effect of taking into account the CoP.

![largeobjectresults03](https://cloud.githubusercontent.com/assets/10450269/20220576/602841b8-a82e-11e6-82f9-f87a8889bc85.png)

**Comment by nawidj** [comment]
Some of the things I noticed during collection. Once the model has a bad point, it takes it very long to recover from it, for example the dent at length 40, width 75 should have been on the object. It took the robot a while to explore this part of the object.

The second thing I noticed is that sometimes the classifier would not converge. This resulted in oversampling location at length 100, width 75.
